pallets/frequency-tx-payment/src/capacity_stable_weights.rs

```diff
@@ -61,6 +61,13 @@ pub trait WeightInfo {
 	// Messages
 	fn add_onchain_message(n: u32, ) -> Weight;
 	fn add_ipfs_message() -> Weight;
+	// Stateful-storage
```

Comment:
This seems to be a bit tricky, how would you know what stable weights are correct? Weights can not be assumed to be stable and I'm not sure what is the consequences for  relying on something like this

Reply:
I'll be working on putting something to address some of your questions tomorrow.